coralproject/talk
# Update i18n config to support new Danish translation

## What does this PR do?

Completes support for Danish.

## How do I test this PR?

- Open Talk
- Change your local storage language to be `da_DK`
- See Talk in Danish!
